openmessaging-api/src/main/java/io/openmessaging/api/order/GenericMessageOrderListener.java

```diff
@@ -26,7 +27,7 @@
  * @version OMS 2.0.0
  * @since OMS 2.0.0
  */
-public interface GenericMessageOrderListener<T> {
+public interface GenericMessageOrderListener<T> extends GenericListener<T> {
```

Comment:
How about GenericOrderedMessageListener?

Reply:
I agree with this opinion, but there is already a interface called 'MessageOrderListener', so i chose to add a 'Generic' prefix